core/capabilities/remote/trigger_publisher.go

```diff
@@ -178,7 +183,8 @@ func (p *triggerPublisher) Receive(_ context.Context, msg *types.MessageBody) {
 			p.lggr.Errorw("failed to register trigger", "capabilityId", p.capInfo.ID, "workflowId", req.Metadata.WorkflowID, "err", err)
 		}
 	} else {
-		p.lggr.Errorw("received trigger request with unknown method", "method", SanitizeLogString(msg.Method), "sender", sender)
+		p.lggr.Errorw("received trigger request with unknown method",
```

Comment:
nit: change to "received message with unknown method".
Also you can potentially catch method == MethodTriggerEvent - that's the most probably case of a bounced-back message